src/test/libcephfs/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if(${WITH_CEPHFS})
     acl.cc
     main.cc
     deleg.cc
+    monaddrsfromcct.cc
```

Comment:
could use a better name than this. it's like `hellojoehowisitgoingwelliamgood`.

Reply:
Done (I hope)